channels/channels.go

```diff
@@ -57,6 +57,7 @@ type Page struct {
 	Tag            string   `json:"tag,omitempty"`
 	Status         Status   `json:"status,omitempty"`
 	Group          string   `json:"group,omitempty"`
+	NoParentGroup  bool     `json:"no_parent_group,omitempty"`
```

Comment:
Let's reuse `Group` instead. We need to change it to the pointer, where nil indicates no interest in this query param, an empty value indicates no parent, and a non-empty value indicates a desired parent. We can also use NullableString here, but let's not do it in this PR.